presto-main/src/main/java/io/prestosql/type/UuidOperators.java

```diff
@@ -44,11 +46,21 @@
 import static io.prestosql.spi.function.OperatorType.NOT_EQUAL;
 import static io.prestosql.spi.function.OperatorType.XX_HASH_64;
 import static io.prestosql.type.UuidType.UUID;
+import static java.util.UUID.randomUUID;
 
 public final class UuidOperators
 {
     private UuidOperators() {}
 
+    @Description("generates a random UUID")
+    @ScalarFunction(deterministic = false)
+    @SqlType(StandardTypes.UUID)
+    public static Slice uuid()
```

Comment:
You could add this in `UuidFunctions` (a new class). No strong opinion.

Reply:
nvrm

Reply:
I briefly considered that but decided it was overkill for one function and we don't expect to have more anytime soon.